contracts/core/Registry.sol

```diff
@@ -2,11 +2,11 @@
 pragma solidity 0.8.19;
 
 import "./IRegistry.sol";
-import {AccessControl} from "@openzeppelin/access/AccessControl.sol";
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
 import {Metadata} from "./libraries/Metadata.sol";
 import "./libraries/Transfer.sol";
 import "./libraries/Native.sol";
-import {ERC20} from "@solady/tokens/ERC20.sol";
+import {ERC20} from "solady/src/tokens/ERC20.sol";
```

Comment:
Hey, I'm not fussed over which format we use, but we should be consistent. In some places we're using the `@` and in others we're not (i.e. here we are for Open Zeppelin but are not for Solady).

Reply:
I guess I have a slight preference for the `@` and using shorter paths (i.e. mapping `@solady` directly to the `src` directory)

Reply:
Same with `@openzeppelin` to the `contracts/` directory